enterprise/suppress-user/factory.go

```diff
@@ -115,14 +117,17 @@ func (m *Factory) Setup(ctx context.Context, backendConfig backendconfig.Backend
 				}
 				repo.Switch(fullRepo)
 				latestSyncCancel()
+				m.Log.Info("Full sync started")
```

Comment:
```suggestion
				repo.Switch(fullRepo)
				m.Log.Info("Switched to full suppression repository")
				latestSyncCancel()
```


We also need a log statement around lines 110-112 to know when the 1st full sync started and ended
```go
m.Log.Info("First full suppression sync started")
...
m.Log.Info("First full suppression sync done")
```

Reply:
We also need a log statement around lines 110-112 to know when the 1st full sync started and ended

```go
m.Log.Info("First full suppression sync started")
...
m.Log.Info("First full suppression sync done")
```

Reply:
added.